RELEASE.md

```diff
@@ -23,16 +23,68 @@ This page contains the release notes for AMD ROCm Software.
 
 #### OS and GPU support changes
 
-* AMD Instinct™ MI300A and MI300X support has been added
+* AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported OSes excluding RHEL9.
```

Comment:
expand OSes to operating systems. 

Recommended
AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported operating systems, excluding RHEL9.

Reply:
Please use the suggestion feature to directly propose edits in the future, its the +- icon